### PR TITLE
Adding Project Sakura logo and detection temporary

### PR DESCRIPTION
**Pull request by hmtheboy154**
Since the logo is in gradient color I think it's good to use png instead of xml. Also the reason I said "temporary" is because since PS is a Lineage-based rom so I check it with `ro.lineage.build.version` , we need to find a way to distingush between LineageOS and Project Sakura.

### CODE REVIEW

File: app/src/main/java/com/farmerbb/taskbar/ui/TaskbarController.java [hunk 1]
```diff
@@ -455,8 +455,10 @@ void drawStartButton(Context context, ImageView startButton, SharedPreferences p
             case PREF_START_BUTTON_IMAGE_APP_LOGO:
                 Drawable drawable;
 
-                if(U.isAndroidGeneric(context)) {
+                if(U.isBlissOs(context)) {
                     drawable = ContextCompat.getDrawable(context, R.drawable.tb_bliss);
+            }   else if(U.isProjectSakura(context)) {
```
Comment (utzcoz) [inline app/src/main/java/com/farmerbb/taskbar/ui/TaskbarController.java:460]: What about aligning your }?

File: app/src/main/java/com/farmerbb/taskbar/util/U.java [hunk 2]
```diff
@@ -1111,8 +1111,20 @@ public static boolean isBlissOs(Context context) {
                 && isSystemApp(context);
     }
 
+    public static boolean isProjectSakura(Context context) {
+        boolean validSakuraBuildProp = false;
+
+        String sakuraVersion = getSystemProperty("ro.lineage.build.version");
+        if(sakuraVersion != null && !sakuraVersion.isEmpty())
```
Comment (utzcoz) [inline app/src/main/java/com/farmerbb/taskbar/util/U.java:1118]: What about adding { and } for if?

Comment (hmtheboy154) [reply app/src/main/java/com/farmerbb/taskbar/util/U.java:1118]: Can you help me algining it ?

Comment (utzcoz) [reply app/src/main/java/com/farmerbb/taskbar/util/U.java:1118]: I think it's better you changed it if you think it is deserved. My review suggestion is only suggestion. You can omit it, if @farmerbb  think current changes are fine.

Comment (farmerbb) [reply app/src/main/java/com/farmerbb/taskbar/util/U.java:1118]: @hmtheboy154 I can fix the brace issues once this is merged

**Comment by farmerbb** [comment]
@hmtheboy154 FYI, `app/src/main/res/drawable/sakura.png` isn't actually needed as the code references `app/src/main/res/drawable-*/tb_sakura.png` instead, so this can be deleted.

**Pull request merged**